### DISCUSSION TITLE
Adding SMTP server failed

### DISCUSSION THREAD

**Discussion by chrismb23** [Budibase/budibase#8457] [Help] [answered]
I'm trying to add my SMTP server to the settings, but the following error keeps appearing:

Failed to save email settings, reason: 139649975323464:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:331:

Is this something I can fix myself?

**Comment by andz-bb** [comment, marked as answer]
hey @chrismb23, I haven't seen that error before but I was able to replicate it on my end when using the wrong port for an SSL SMTP setup. In my case I'm using SendGrid and they needed me to use port 465 for SSL connections. Could be worth double checking that depending on what SMTP provider you're using. Hope that helps!
